common4j/src/main/com/microsoft/identity/common/java/commands/SilentTokenCommand.java

```diff
@@ -109,14 +108,20 @@ public AcquireTokenResult execute() throws Exception {
                 } catch (UiRequiredException | ClientException e) {
```

Comment:
What about other types of Exceptions? I believe you want those to be returned from first controller as well? If yes, then you need a catch block for generic `Exception` and add the same logic there

Reply:
I see what you mean. let me check

Reply:
done

Reply:
Is this fully addressed? I think you need to check for ii == 0 in the generic Exception block but I don't see that happening

Reply:
Discussed offline; lgtm

Reply:
yes, this is addressed